cads_adaptors/adaptors/cadsobs/adaptor.py

```diff
@@ -1,16 +1,21 @@
-import logging
 import tempfile
 from pathlib import Path
 
 from cads_adaptors.adaptors.cadsobs.api_client import CadsobsApiClient
 from cads_adaptors.adaptors.cds import AbstractCdsAdaptor
 from cads_adaptors.exceptions import CadsObsRuntimeError
 
-logger = logging.getLogger(__name__)
-
 
 class ObservationsAdaptor(AbstractCdsAdaptor):
     def retrieve(self, request):
+        try:
+            output = self._retrieve(request)
+        except Exception as e:
+            self.context.add_user_visible_error(repr(e))
+            raise
```

Comment:
```suggestion
            raise e
```
Please raise the Exception, it will be used for filtering error events in the monitoring. We have a custom excpetions for CadsObs

Reply:
fixed in c31e796